pages/project-version/update/endorse/content/index.js

```diff
@@ -93,9 +93,6 @@ module.exports = {
     }
   },
   buttons: {
-    submit: {
-      application: 'Submit {{#canEndorse}}to Home Office{{/canEndorse}}{{^canEndorse}}PPL application{{/canEndorse}}',
-      amendment: 'Submit PPL amendment'
-    }
+    submit: 'Submit {{^isApplication}}PPL {{type}}{{/isApplication}}{{#isApplication}}{{#canEndorse}}to Home Office{{/canEndorse}}{{^canEndorse}}PPL application{{/canEndorse}}{{/isApplication}}'
```

Comment:
whoagif.gif

Reply:
yeah, I was actually going to paste this in slack for lolz, I mean, can't the button just say "submit"?